<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeeklyScheduleAdd</class>
 <widget class="QDialog" name="WeeklyScheduleAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Add_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>원하는 요일</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>시작 시간</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>끝 시간</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>할 일</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="WeekSelect"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="WeekStartTime">
        <property name="font">
         <font>
          <family>HY견고딕</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="WeekEndTime">
        <property name="font">
         <font>
          <family>HY견고딕</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="WeekContents">
        <property name="font">
         <font>
          <family>굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
